content/concepts/content-management/02-content-models.md

```diff
@@ -6,20 +6,47 @@ metaDescription: "sensenet Content Model and Schema"
 
 sensenet organizes content into content repositories, that allows you to group all the related content for a project together. Since everything is a content is sensenet this repository includes documents, workspaces, users, settings, workspaces and many more.
 
-Each repository has a content model, a schema that represents the content types you create.
+Each repository has a content model, a schema that represents the content types you create. This is project-specific, so it is your own. It is important defining the business logic and creating the content types at the beginning of the project.
 
 # Content type hierarchy
```

Comment:
nagyon hirtelen belevágunk a közepébe itt, jó lenne (ha ez megoldható) valamiféle intro ehhez a részhez. Nem érthetőek az alapok nekem itt.